fs/littlefs/Kconfig

```diff
@@ -4,3 +4,13 @@ config FS_LITTLEFS
 	depends on !DISABLE_MOUNTPOINT
 	---help---
 		Build the LITTLEFS file system. https://github.com/ARMmbed/littlefs.
+
+config FS_LITTLEFS_BLOCK_FACTOR
+	int "LITTLEFS Block size multiple factor"
+	default 4
+	depends on FS_LITTLEFS
+
+config FS_LITTLEFS_BLOCK_CYCLE
```

Comment:
This config also depends on `FS_LITTLEFS`.
Wrap these two in a outer `if/endif` block to avoid spreading `depends on` statements.